components/recipe/video-lesson.tsx

```diff
@@ -19,13 +19,16 @@ const Sponsorships = styled.p`
   }
 `;
 
-const VideoLesson = ({ youtubeEmbedId }: { youtubeEmbedId: string }) => {
+const VideoLesson: FC<{ youtubeEmbedId: string }> = ({ youtubeEmbedId }) => {
   const [vidHeight, setVidHeight] = useState(450);
 
   useEffect(() => {
-    const containerWidth = document.getElementById('yt-embed').offsetWidth;
-    const proportionedHeight = (containerWidth * 9) / 16;
-    setVidHeight(proportionedHeight < 400 ? proportionedHeight : 400);
+    const container = document.getElementById('yt-embed');
+    if (container) {
```

Comment:
We should assume it's there. Perhaps we do `document.getElementById('yt-embed')!!.offsetWidth` if that helps.

Reply:
Ok. This solution is really better.